docs/docs/advanced-use/custom-fields.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Custom Fields
+---
+
+:::info
+
+Custom fields were introduced in Flagsmith [2.116.0](https://github.com/Flagsmith/flagsmith/releases/tag/v2.116.0).
+
```

Comment:
We should add in here what type of FS license you need to get access to custom fields.

Reply:
Added in bea277caa2fd13e8810b84b7fa473abc47dea560